app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= content_tag :h1, "Welcome To Viewing Party!" %>
+<%= content_tag :h3, "This website allows you to create a 'viewing party' with your friends to watch movies together online from the comfort of your own homes. Especially during these hard time during a pandemic when it isn't safe to be in the same room." %>
+
+
+<%= form_tag "/login" do %>
```

Comment:
Like we discussed, possibly a refactor opportunity to use form_for or form_with instead of form_tag here.

Reply:
or form_with...? Aren't they replacing form_tag with that?